cmd/hatchet-migrate/migrate/migrations/20251224145920_v1_0_65.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+
+ALTER TYPE v1_event_type_olap ADD VALUE 'COULD_NOT_SEND_TO_WORKER';
+
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+
+ALTER TYPE v1_event_type_olap DROP VALUE 'COULD_NOT_SEND_TO_WORKER';
```

Comment:
hehe, this one won't work - there's no `DROP VALUE`, so to have a down migration here we'll need to create a new enum, cast the type of the column, drop, and rename like you had before. The hope is we never need to actually run the down migration...